mythril/analysis/modules/dos.py

```diff
@@ -70,7 +70,7 @@ def _analyze_states(self, state: GlobalState) -> List[Issue]:
 
                 try:
                     self._jumpdest_count[transaction][target] += 1
-                    if self._jumpdest_count[transaction][target] == 4:
+                    if self._jumpdest_count[transaction][target] == 3:
```

Comment:
We should use that constant here right? Propogating that to here might be a problem.

Reply:
Unfortunately there's currently no way to pass arguments to modules or access svm variables from it. I'd leave this as a hardcoded value for now. Later, we could have Mythril set a ModuleConfigAnnotation as a new feature?